api.md

```diff
@@ -0,0 +1,115 @@
+# cozysdk-client
```

Comment:
As a remainder, we have some guidelines for writing docs: https://github.com/cozy-labs/cozy-dev-docs/blob/master/GUIDELINES.md